packages/ppx_deriving_cmdliner/ppx_deriving_cmdliner.0.4.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "Seb Mondet <seb@mondet.org>"
+authors: [
+  "Isaac Hodes <isaachodes@gmail.com>"
+  "B. Arman Aksoy <arman@aksoy.org>"
+  "Seb Mondet <seb@mondet.org>"
+  "Nick Zalutskiy <nick@const.fun>"
+  "Armaël Guéneau <armael.gueneau@ens-lyon.fr>"
+]
+homepage: "https://github.com/hammerlab/ppx_deriving_cmdliner"
+bug-reports: "https://github.com/hammerlab/ppx_deriving_cmdliner/issues"
+dev-repo: "git+https://github.com/hammerlab/ppx_deriving_cmdliner.git"
+doc: "http://hammerlab.github.io/ppx_deriving_cmdliner"
+license: "MIT"
+version: "0.4.1"
+tags: ["syntax" "cli"]
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+run-test: [
+  ["jbuilder" "runtest" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03"}
+  "cmdliner" {>= "1.0.0"}
+  "result"
+  "ppx_deriving" {>= "4.0" & < "5.0"}
+  "dune"         {build}
+  "ocamlfind"    {build}
+  "ppxfind"      {build}
+  "cppo"         {build}
+  "alcotest"     {with-test}
+  "ppx_import"   {with-test & >= "1.1"}
+]
+synopsis: "Cmdliner.Term.t generator"
+description: """
+ppx_deriving_cmdliner is a ppx_deriving plugin that generates
+a Cmdliner Term.t for a record type."""
```

Comment:
Just an FYI, but the description ends up getting folded into the opam web pages, and if I were reading through them, I'm not sure this would be enough for me to figure out why I want to use the package.

Reply:
The package-page also links to the webpage that has properly formatted examples and all.
This description also shows up in terminals (`opam show ...`).

PRs welcome in any case.

Reply:
I don't think most people would paste the URL and look at the page if they don't already know that they need something. There are a truly huge number of packages in OPAM, and one doesn't have time to do that all day long. I think it's pretty important that a package explain what it's for, especially so people who are skimming descriptions can find the right one.

And, as I don't know what this package is for, I can't submit a PR because to do that better, as I'd first have to know what to say.